### ISSUE TITLE
I am a professional storyteller/videographer, seeking an animator/designer and a math expert.

### ISSUE THREAD

**Issue by cagrimertbakirci** [leios/SoME_Topics#131]
I am a Turkish popular science creator. See videos here: https://www.youtube.com/evrimagaci

I've done English videos too in the past, they were less professional (since I was less professional at the time) but so that you can see my skills in English, here's an example: https://www.youtube.com/watch?v=8rkKfEk88zg (I am the male one, just FYI :P)

Okay, so I can probably find a topic on my own (I have a PhD in Mechanical Engineering with a minor in Biology), so I'd love to collaborate with an animator. But I would also love to take the expertise of a mathematician and convert their idea to a story and produce it.

I can do the video edits, cuts, text overlay, videography, recording, scripting and all that.

In other words:

**Option 1:** I need an animator. **OR**
**Option 2:** I need an animator _and_ a math expert.

We can make either one work.

Let's do this!
